src/components/ContactActionsPopover.vue

```diff
@@ -0,0 +1,214 @@
+<template>
+  <ion-content>
+    <ion-list>
+      <ion-list-header>{{ translate(placeholder) }}</ion-list-header>
+      <ion-item button @click="copyInfo(value)">
+        {{ translate("Copy") }}
+      </ion-item>
+      <ion-item @click="updateContactField(type)" button>
+        {{ translate("Edit") }}
+      </ion-item>
+      <ion-item @click="deleteContactField(type)" button lines="none">
```

Comment:
No need to pass type from here, we can simply access type inside method as we have passed it as prop.